### PR TITLE
Switching some [vector: any] to [same as input].. but not only

### PR DESCRIPTION
**Pull request by DelazJ**
Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorgeneral.rst [hunk 1]
```diff
@@ -176,33 +176,29 @@ Parameters
...
+     - ``INPUT``
+     - [vector: any]
+     - The input layer
+   * - **Fields**
+     - ``FIELDS``
+     - [tablefield: any] [list]
+     - Fields to match duplicates by
+   * - **Filtered (no duplicates)**
+     - ``OUTPUT``
+     - [feature sink]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeneral.rst]: @havatv forgive my ignorance but are these not layers? I saw the parameter type in the code and don't know what a feature sink means but for me the output is the same as if i had used any of the common algs, eg centroid

File: source/docs/user_manual/processing_algs/qgis/vectorgeneral.rst [hunk 2]
```diff
@@ -213,32 +209,28 @@ Outputs
...
+     - ``RETAINED_COUNT``
+     - [Number]
+     - Count of retained records
+   * - **Count of discarded duplicate records**
+     - ``DUPLICATE_COUNT``
+     - [Number]
+     - Count of discarded duplicate records
+   * - **Filtered (no duplicates)**
+     - ``OUTPUT``
+     - [String]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorgeneral.rst]: same would apply here

**Comment by DelazJ** [comment]
> I hope you won't touch the documents that are modified in my "in progress" PRs.

My changes are self contained. I only update table-like algorithms so, as long as you don't (re)touch an alg that had been moved to the table structure, there should be no conflict. And anyway, "`git rebase` your friend is" ;)

**Comment by havatv** [comment]
> > I hope you won't touch the documents that are modified in my "in progress" PRs.
> 
> My changes are self contained. I only update table-like algorithms so, as long as you don't (re)touch an alg that had been moved to the table structure, there should be no conflict. And anyway, "`git rebase` your friend is" ;)

Thanks!

File: source/docs/user_manual/processing_algs/qgis/vectoranalysis.rst [hunk 3]
```diff
@@ -1072,16 +1072,16 @@ Outputs
...
-      - The output layer with additional fields reporting the
-        overlap (in map units and percentage) of the input feature
-        overlapped by each of the selected layers.
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Output layer**
+     - ``OUTPUT``
+     - [vector: any]
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectoranalysis.rst]:
```suggestion
     - [same as input]
```

**Pull request merged**